modules/apps/frontend-data-set/frontend-data-set-admin-web/src/main/resources/META-INF/resources/js/data_set/actions/Actions.tsx

```diff
@@ -242,17 +242,10 @@ const Actions = ({dataSet, namespace, spritemap}: IDataSetSectionProps) => {
 	};
 
 	const updateActionsOrder = async ({order}: {order: string}) => {
-		let actionTypeOrder =
+		const actionTypeOrder =
 			activeTab === 0 ? 'fdsItemActionsOrder' : 'fdsCreationActionsOrder';
 
-		let apiURL = API_URL.DATA_SETS;
-
-		if (Liferay.FeatureFlags['LPD-15729']) {
-			actionTypeOrder =
-				activeTab === 0 ? 'itemActionsOrder' : 'creationActionsOrder';
```

Comment:
@markocikos I cannot find the related name change in the FDSViewsPortlet (from https://github.com/liferay/liferay-portal/commit/17830a029910d824be2212be6b25dddeee485ec5)

Any other order property follows the `fds*Order` convention.

![image](https://github.com/user-attachments/assets/bfe84aed-5cc2-4664-82fb-b31ddef37c7f)

Reply:
Yeah, the change in PR is fine. This will be an intended rename in https://github.com/liferay-frontend/liferay-portal/pull/4450. At one point I intended to reduce hierarchy along with renaming, but I needed to split PR and remove rename. This stayed in by mistake.